### PR TITLE
Feature game setup

### PR DESCRIPTION
**Pull request by amsmyth1**
Created board_setup class and test code
Created readable attributes: board, computer_cruiser, player_cruiser, player_sub, game_messages
Created methods: computer_select_cells, computer_place_ship, player_valid_entry, player_place_ship

### CODE REVIEW

File: lib/board_setup.rb [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    @game_messages = GameMessage.new
+
+  end
+
+  def computer_select_cells(length)
+    randomized_cells = @board.cells.keys.shuffle
+    spaces = randomized_cells.shift(length)
+  end
+
+  def computer_select_cells_submarine
```
Comment (jbranham1) [inline lib/board_setup.rb]: Since we combined cruiser and submarine into one method I'm not sure we need this one anymore! What do you think?

**Review by jbranham1 — CHANGES_REQUESTED** [review]
Besides removing one un-used method, this all looks SO good and well though out. Good job!

File: lib/board_setup.rb (hunk 1 above)
Comment (amsmyth1) [reply lib/board_setup.rb]: I agree, I removed it from the board_setup file and confirmed all tests still pass.

**Comment by amsmyth1** [comment]
Changes have been made, this is ready to merge!

**Review by jbranham1 — APPROVED** [review]
Send it!

**Pull request merged**